### PR TITLE
Android support

### PR DESCRIPTION
**Pull request by adrum**
This is a pull request adding Android support for issue #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ lane :test do
...
 end
+
+# or
+
+lane :android do
+  appicon(appicon_image_file: 'spec/fixtures/Themoji.png',
+    appicon_devices: [:android],
+    appicon_res:'app/res',
+    appicon_res_foldername:'mipmap',
+    appicon_res_filename:'ic_launcher',
```
Comment (KrauseFx) [inline README.md]: Trailing comma should be removed

File: README.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ lane :test do
...
+    )
```
Comment (KrauseFx) [inline README.md]: Please move this to the far left

File: fastlane/Fastfile [hunk 3]
```diff
@@ -2,3 +2,8 @@ lane :test do
   appicon(appicon_image_file: 'spec/fixtures/Themoji.png',
     appicon_devices: [:ipad, :iphone])
 end
+
+lane :android do
+  appicon(appicon_image_file: 'spec/fixtures/Themoji.png',
+    appicon_devices: [:android])
+end
```
Comment (KrauseFx) [inline fastlane/Fastfile]: Missing new line at the end of the file

File: lib/fastlane/plugin/appicon/actions/appicon_action.rb [hunk 4]
```diff
@@ -90,12 +126,30 @@ def self.available_options
...
+                             default_value: 'ic_launcher',
+                               description: "The output filename of each image (Android only)",
+                                  optional: true,
+                                      type: String),
         ]
       end
 
       def self.is_supported?(platform)
-        [:ios, :mac, :macos, :caros].include?(platform)
+        [:ios, :mac, :macos, :caros, :android].include?(platform)
```
Comment (KrauseFx) [inline lib/fastlane/plugin/appicon/actions/appicon_action.rb]: `:caros` cc @neonichu 😂

**Review by KrauseFx — CHANGES_REQUESTED** [review]

**Comment by KrauseFx** [comment]
Hey @adrum, thanks for the PR. I added some comments to the documentation. Thinking about this, I'd prefer the Android part to be its own action within this plugin. To do that, duplicate the action in this directory (https://github.com/KrauseFx/fastlane-plugin-appicon/tree/master/lib/fastlane/plugin/appicon/actions) and name it `android_appicon`. This way we don't end up with so many possible parameters 👍

**Comment by adrum** [comment]
Hey @KrauseFx, that's a great idea to separate android into its own action. I ended up only adding one parameter to the new android action when compared to the original action. I actually ran `rubocop -a` this time, too.

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
Looks great, thanks 👍

**Comment by joshdholtz** [comment]
@adrum Thanks for adding this to @KrauseFx's already awesome plugin ❤️

**Pull request merged**